src/main/java/core/basesyntax/model/Bet.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.model;
+
+public class Bet {
+    private int value;
+    private double risk;
+
+    public Bet(int value, double risk) {
```

Comment:
add check for negative numbers.
For example 1XBET we can't put the bet with a negative value :)